File: .sdk/tm/php/utility/MakeContext.php
<?php
declare(strict_types=1);

// DymoApiIntroduction SDK utility: make_context

require_once __DIR__ . '/../core/Context.php';

class DymoApiIntroductionMakeContext
{
    public static function call(array $ctxmap, ?DymoApiIntroductionContext $basectx): DymoApiIntroductionContext
    {
        return new DymoApiIntroductionContext($ctxmap, $basectx);
    }
}

File: rb/utility/make_context.rb
# DymoApiIntroduction SDK utility: make_context
require_relative '../core/context'
module DymoApiIntroductionUtilities
  MakeContext = ->(ctxmap, basectx) {
    DymoApiIntroductionContext.new(ctxmap, basectx)
  }
end
